src/common/viz/ConfigDialog.js

```diff
@@ -200,7 +200,12 @@ define([
             settings[namespace] = {};
 
             for(const name of Object.keys(this._widgets[namespace])){
-                const value = await this._widgets[namespace][name].getValue();
+                let value;
+                try {
+                    value = await this._widgets[namespace][name].getValue();
+                } catch (e) {
+                    this._dialog.modal('hide');
```

Comment:
This will actually result in hiding errors (ie, it will fail silently). It would be better to notify the user of the failure (esp since it could be an issue they can fix such as authentication or invalid volume). I have to look at the code more as there might be a better place to notify the user. You could either show the error in a toast, or use it like a validation message on the form fields of the failing widget itself (probably better).

Reply:
Yeah, I agree that would be better, esp. this dialog is reused in multiple avenues. I think we should stick to a toast for now, and have this form error notification for the dialog in a separate PR.

Reply:
In that case, can you re-throw the error after hiding the modal so it at least shows up in the dev console?